SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -295,6 +295,26 @@ bool USpatialPackageMapClient::SerializeObject(FArchive& Ar, UClass* InClass, UO
 	return true;
 }
 
+const FClassInfo* USpatialPackageMapClient::TryResolveNewDynamicSubobjectAndGetClassInfo(UObject* Object, Worker_EntityId EntityId, USpatialClassInfoManager* ClassInfoManager)
+{
+	FUnrealObjectRef Ref = GetUnrealObjectRefFromObject(Object);
+	if (Ref.IsValid())
+	{
+		UE_LOG(LogSpatialPackageMap, Error, TEXT("Trying to resolve a dynamic subobject twice! Object %s, EntityId %d."), Object ? *Object->GetName() : TEXT("null"), EntityId);
```

Comment:
Can use GetNameSafe() here instead.

Reply:
Can you add a comment about why we return nullptr here, instead of the existing FClassInfo.

Reply:
Have re-written the function a bit, can you check the logic again?